janusgraph-test/src/test/java/org/janusgraph/core/attribute/JtsGeoshapeHelperTest.java

```diff
@@ -16,11 +16,20 @@
 
 import org.janusgraph.core.attribute.GeoshapeHelper;
 import org.janusgraph.core.attribute.JtsGeoshapeHelper;
+import org.junit.Assert;
```

Comment:
We seem to typically use `import static org.junit.Assert.assertEquals;` in this repo, so that we can write code such as:

```java
assertEquals(...);
```

instead of the longer

```java
Assert.assertEquals(...);
```

Can you make the change to use the shorter form? It doesn't matter much in this specific case, since you only use the assert once, but it will make future code easier to write if they follow the same pattern.

Reply:
ok. Thanks for your review.